jpo-ode-svcs/src/test/resources/application.yaml

```diff
@@ -3,6 +3,20 @@ spring:
     multipart:
       max-file-size: 1MB
       max-request-size: 1MB
+  embedded:
+    kafka:
+      brokers:
+        property: localhost:9092
```

Comment:
Why isn't this referencing port 4242?

Reply:
It's a holdover from a prior implementation. it does not affect the test setup or anything, so it should be removed. I'll do that in the next changeset so that these changes can be merged without needing another review. I'll make sure to call this comment out in the incoming PR